File: MDEV1001ICE7B/app/src/main/java/com/example/mdev1001_ice7b/Movie.kt
package com.example.mdev1001_ice7b

data class Movie(
    val movieID: String,
    val title: String,
    val studio: String,
    val genres: List<String>,
    val directors: List<String>,
    val writers: List<String>,
    val actors: List<String>,
    val year: Int,
    val length: Int,
    val shortDescription: String,
    val mpaRating: String,
    val criticsRating: Double
)

File: ICE9A/ICE9A/Movie.swift
//
//  Movie.swift
//  ICE9A
//
//  Created by Dhrumil Jigneshkumar Desai on 2023-07-21.
//

import Foundation

struct Movie: Codable
{
    let _id: String
    let movieID: String
    let title: String
    let studio: String
    let genres: [String]
    let directors: [String]
    let writers: [String]
    let actors: [String]
    let year: Int
    let length: Int
    let shortDescription: String
    let mpaRating: String
    let criticsRating: Double
}
